WORKSPACE

```diff
@@ -19,6 +19,13 @@ workspace(name = "oak")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
+http_archive(
+    name = "com_google_absl",
+    urls = ["https://github.com/abseil/abseil-cpp/archive/27c30ec671cb7b5ba84c4e79feff7fd0b0ac6338.zip"],
```

Comment:
Perhaps also add a comment similar to https://github.com/google/asylo/blob/master/asylo/bazel/asylo_deps.bzl#L152 , so we have a hint of roughly when to update (for lack of proper version numbers).

Reply:
Done.